planner_name: A*
path_topic: planned_path
env_file: env1.yaml